rfcs/tag-component.md

```diff
@@ -0,0 +1,61 @@
+# Tag Component
+
+### Exports
+
+- `Tag`
+- `StyledRoot`
+- `StyledCross`
+
+### `Tag` API
+
+- `color: STYLE.blue(theme.colors.primary470) | STYLE.orange(theme.colors.warning400) | STYLE.green(theme.colors.positive400) | STYLE.red(theme.colors.negative400)`:
+  the color theme to be applied to a Tag. Default is `STYLE.blue`.
+- `children: React$Node`:
+  Component or String value for label of tag. Default is empty string
+- `disabled: boolean`:
+  Disable control from being changed
+- `overrides: {Root: (props: {[string]: any}) => React$Node, Cross: (props: {[string]: any}) => React$Node}`
+  - Custom `Cross` button.
+  - `Root` wrapper element for the whole tag control to apply styles
+- `onCrossClick: func(remove => {})`:
+  handler for events on Cross button element. `remove` argument method can be called any time to remove tag.
```

Comment:
How were you imagining `remove` works? I don't think the tag can remove itself, it's up to the parent component to remove the tag. 

Maybe we should just change this to `onActionClick: (evt) => void`?

Reply:
technically it's possible, but raises concerns. I will remove this
http://codetheory.in/remove-a-mounted-react-component-with-react-unmountcomponentatnode/